assignment_1.py

```diff
@@ -0,0 +1,251 @@
+Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
+Type "help", "copyright", "credits" or "license()" for more information.
+#list built in function
+list=[ 'banana' ,'graps' ,'apple']
+#append
+list.append("lemon")
+print(list)
+"""
+['banana', 'graps', 'apple', 'lemon']
+"""
+#copy
+fruits=list.copy()
+print(fruits)
+"""
+['banana', 'graps', 'apple', 'lemon']
+"""
+"""
+#count
+fruits.count()
+Traceback (most recent call last):
+  File "<pyshell#9>", line 1, in <module>
+    fruits.count()
+TypeError: list.count() takes exactly one argument (0 given)
+list.count()
+Traceback (most recent call last):
+  File "<pyshell#10>", line 1, in <module>
+    list.count()
+TypeError: list.count() takes exactly one argument (0 given)
+fruits.count(apple)
+Traceback (most recent call last):
+  File "<pyshell#11>", line 1, in <module>
+    fruits.count(apple)
+NameError: name 'apple' is not defined. Did you mean: 'tuple'?
+fruits.count('apple')
+1
+"""
+#extend
+vegetables=['onion','carrot','ginger']
+fruits.extend(vegetables)
+print(fruits)
+"""
+['banana', 'graps', 'apple', 'lemon', 'onion', 'carrot', 'ginger']
+"""
+"""
+#index
+fruits.index()
+Traceback (most recent call last):
```

Comment:
Files should not contain these errors. Use VS Code to write code. So these errors will be rectified when you run the code.